### PR TITLE
2 feature create mock data for events

### PR DESCRIPTION
**Pull request by akajugz**

### CODE REVIEW

File: src/data/data.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import { MockEvent } from "../types/MockEvents";
+
+export const mockEvents: MockEvent[] = [
+  {
```
Comment (Charlie-robin) [inline src/data/data.tsx:4]: There is some data on Figma to use for this. 

image

File: src/data/data.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
+import { MockEvent } from "../types/MockEvents";
+
+export const mockEvents: MockEvent[] = [
```
Comment (Charlie-robin) [inline src/data/data.tsx:3]: Can you change the name of the file from data to mockEvents.ts please

File: src/data/data.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    date: "2023-08-15",
+    description:
+      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit officiis consequuntur quas qui fuga minus iusto distinctio eum, ratione hic accusamus minima vitae quisquam libero nostrum et totam voluptates suscipit?",
+    images: [
+      "https://www.goodwood.com/globalassets/.road--racing/event-coverage/fos/2020/3-march/post/goodwood-festival-of-speed-2020-postponed-m-sport-ford-fiesta-wrc-fos-2019-drew-gibson-goodwood-26032020.jpg?crop=(0,270,2600,1733)&width=1600",
+      "https://www.throttle-blips.com/wp-content/uploads/2019/07/Goodwood-FOS-2017-1-1500x1000.jpg",
+      "https://cf-images.us-east-1.prod.boltdns.net/v1/static/6109416600001/30fcfe57-dab2-4731-9525-1fe63bf9fea6/3ad7ba6d-6f5d-4e8f-b025-856b53a8f0f0/1280x720/match/image.jpg",
+    ],
+  },
+  {
```
Comment (Charlie-robin) [inline src/data/data.tsx:16]: Can you add id keys to each object

File: src/types/MockEvents.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export type MockEvent = {
+  name: string;
```
Comment (Charlie-robin) [inline src/types/MockEvents.ts:2]: Also add a id key to the type as well

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Some points to change before mergeing

File: src/data/data.tsx (hunk 2 above)
Comment (Charlie-robin) [reply src/data/data.tsx:3]: It is still `data.tsx` can you change it to `mockEvents.ts` please

File: README.md [hunk 5]
```diff
@@ -60,6 +60,10 @@ const fakeStyle = {
     ```
 ````
 
+## Mock Data
+
+- The data has been added for the project.
```
Comment (Charlie-robin) [inline README.md:65]: Can you be a bit more specific with this?

**Review by Charlie-robin — APPROVED** [review]
Looks good to me 👍

**Review by olirob93 — APPROVED** [review]

**Pull request merged**